server/startup/migrations/v069.js

```diff
@@ -0,0 +1,14 @@
+RocketChat.Migrations.add({
+	version: 69,
+	up: function() {
+		const settings = RocketChat.models.Settings.find({ _id: /^Accounts_OAuth_Custom_.+/ }).fetch();
```

Comment:
Heh `settings` is actually used, any idea how can we report this bug to @codacy?